patroni/dcs/__init__.py

```diff
@@ -652,6 +652,7 @@ def __init__(self, config):
         self._cluster_valid_till = 0
         self._cluster_thread_lock = Lock()
         self._last_lsn = ''
+        self._dcs_last_seen = 0
```

Comment:
This is exactly what I meant.
Now we just need to set the value every time the `get_cluster()` call succeeds (right after `cluster = self._load_cluster()`)

Reply:
Maybe we also should rename it to the `_last_seen` and the respective property.

Reply:
Done